cmd/spiffe-aws-assume-role/cli/cli_test.go

```diff
@@ -0,0 +1,98 @@
+package cli
+
+import (
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/square/spiffe-aws-assume-role/internal/mocks"
+	"github.com/square/spiffe-aws-assume-role/internal/test"
+	"github.com/square/spiffe-aws-assume-role/pkg/credentials"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+var NoContext *CliContext = nil
+
+func TestParsesSessionDuration(t *testing.T) {
+	args := []string{
+		"credentials",
+		"--session-duration=PT5M",
+		// We only specify the following fields because they're required
+		"--audience=foo",
+		"--role-arn=bar",
+		"--spiffe-id=baz",
+	}
+	context, err := parse(args)
+	require.NoError(t, err)
+
+	cli := context.Model.Target.Interface().(CLI)
+	sessionDuration := cli.Credentials.SessionDuration
+	require.EqualValues(t, 5, sessionDuration.Minutes())
+}
+
+func TestDisplaysTopLevelHelp(t *testing.T) {
+	args := []string{"--help"}
+	require.NoError(t, Run(args, NoContext))
+}
+
+func TestDisplaysCredentialsHelp(t *testing.T) {
+	args := []string{"--help", "credentials"}
+	require.NoError(t, Run(args, NoContext))
+}
```

Comment:
These are just some sanity tests to ensure that we have basic parsing functionality.